handler.go

```diff
@@ -387,12 +388,10 @@ func (ca *Cagent) getVMStatMeasurements(f func(string, MeasurementsMap, error))
 		for _, name := range ca.Config.VirtualMachinesStat {
 			vm, err := vmstat.Acquire(name)
 			if err != nil {
-				log.Warnf("acquire vm provider \"%s\": %s", name, err.Error())
-			} else {
-				if err = vm.IsAvailable(); err != nil {
-					log.Warnf("vm provider \"%s\" either not available or not enabled for host system %s: %s", name, runtime.GOOS, err.Error())
+				if err != vmstattypes.ErrNotAvailable {
+					log.Warnf("acquire vm provider \"%s\": %s", name, err.Error())
```

Comment:
Change `acquire vm provide` into something more human-friendly?
`Error while acquiring vm provider or so`

Also, don't we usually have prefixes for the logs?

Reply:
done